File: app/src/main/java/com/wismna/geoffroy/donext/data/TodayTask.java
package com.wismna.geoffroy.donext.data;

import org.joda.time.LocalDate;

import androidx.room.ColumnInfo;
import androidx.room.PrimaryKey;

public class TodayTask {
    @PrimaryKey()
    public long _id;

    @ColumnInfo(name = "name")
    public String name;

    @ColumnInfo(name = "todaydate")
    public LocalDate todayDate;

    @ColumnInfo(name = "tasklistname")
    public String taskListName;
}

File: app/src/main/java/com/wismna/geoffroy/donext/data/TodayTask.kt
package com.wismna.geoffroy.donext.data

import androidx.room.ColumnInfo
import androidx.room.PrimaryKey
import org.joda.time.LocalDate

class TodayTask {
    @JvmField
    @PrimaryKey
    var _id: Long = 0

    @JvmField
    @ColumnInfo(name = "name")
    var name: String? = null

    @JvmField
    @ColumnInfo(name = "todaydate")
    var todayDate: LocalDate? = null

    @JvmField
    @ColumnInfo(name = "tasklistname")
    var taskListName: String? = null
}
